azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -314,6 +330,17 @@ public void run() {
               lastExecutionDirCleanTime = currentTime;
             }
 
+            if (flowMaxRunningTimeInMins > 0 && currentTime - LONG_RUNNING_FLOW_KILLING_INTERVAL_MS > lastLongRunningFlowCleanTime) {
```

Comment:
Again, I would encourage using a class like `Duration` and objects for handling this rather than using primitives.